[Exchange Online Message Trace] Add Additional Look-back Time & Fix Cursor Value (#8717)
elastic/integrations: packages/microsoft_exchange_online_message_trace/_dev/build/docs/README.md

- The default `Polling Interval` and `Initial Interval` values are configured to `1h`, you can however change these to your required values. The look-back 
  value of `Initial Interval` should not exceed `200 hours` as this might cause unexpected errors with the API.

> - The default `Additional Look-back Time` value is configured for to `1h`. 

Reviewer: ```suggestion
- The default `Additional Look-back Time` value is configured for `1h`. 
```

>   This is intended to allow for catching events that were not initially present because of eventual consistency.

Reviewer: ```suggestion
  This is intended to capture events that may not have been initially present due to eventual consistency.
```
